docker/Makefile

```diff
@@ -0,0 +1,19 @@
+COMPOSE_IMAGES := $(shell docker-compose config 2>/dev/null | awk '{if ($$1 == "image:") print $$2;}')
+
+.PHONY: clean build save 
+
+clean:
+	docker system prune -f --all --volumes
+	rm -rf iml-images.tar.gz
+
+build: clean
+	DOCKER_BUILDKIT=1 docker build -t imlteam/manager-service-base:5.1 -f base.dockerfile ../
+	DOCKER_BUILDKIT=1 docker build -t rust-iml-base -f rust-base.dockerfile ../
+	COMPOSE_DOCKER_CLI_BUILD=1 docker-compose build
+
+save: build
+	COMPOSE_DOCKER_CLI_BUILD=1 docker-compose pull postgres rabbit realtime view-server device-aggregator update-handler
```

Comment:
I'll add in the grafana and influxdb images once I rebase on top of this. Should we also add:
```
iml-agent-comms
iml-warp-drive
iml-action-runner
iml-stratagem
iml-manager-cli
iml-ostpoool
iml-postoffice
```
?

Reply:
This is only to pull things that are built in other repos. All of the above is already created in the `build` task.